multicluster/config/rbac/kustomization.yaml

```diff
@@ -7,8 +7,11 @@ resources:
 - service_account.yaml
 - role.yaml
 - role_binding.yaml
-- leader_election_role.yaml
-- leader_election_role_binding.yaml
+# We disabled leader election in controller, so this role and
```

Comment:
I think we should make it clearer. Do you mean Multi-cluster Controller or antrea-mc-controller?

Reply:
sure, let me change it to Multi-cluster Controller.